src/App.tsx

```diff
@@ -1,22 +1,82 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
-
+import { useSelector, useDispatch } from 'react-redux';
 import './App.scss';
-import { Start } from './components/Start';
+import Loading from './components/Loading/Loading';
+import PostList from './components/PostList/PostList';
+import Button from './components/Button/Button';
+import {
+  loadPosts, getLoaded, getVisiblePosts, getLoading, getMessage, getPosts, getQuery,
+} from './store/index';
+
+import { setQuery } from './store/search';
+
+const App: React.FC = () => {
+  const dispatch = useDispatch();
+  const isLoading = useSelector(getLoading);
+  const errorMessage = useSelector(getMessage) || '';
+  const posts = useSelector(getPosts) || [];
```

Comment:
```suggestion
  const errorMessage = useSelector(getMessage);
  const posts = useSelector(getPosts);
```
У тебя вроде в initialState должны эти значения стоять